laws/src/main/scala/cats/laws/discipline/AlternativeTests.scala

```diff
@@ -19,13 +20,16 @@ trait AlternativeTests[F[_]] extends ApplicativeTests[F] with MonoidKTests[F]  {
     implicit def ArbFA: Arbitrary[F[A]] = ArbF.synthesize[A]
 
     new RuleSet {
-      val name = "alternative"
-      val bases = Nil
-      val parents = Seq(monoidK[A], applicative[A,B,C])
-      val props = Seq(
-        "left distributivity" -> forAll(laws.alternativeLeftDistributivity[A,B](_, _, _)),
-        "right distributivity" -> forAll(laws.alternativeRightDistributivity[A,B](_, _, _)),
-        "right absorption" -> forAll(laws.alternativeRightAbsorption[A,B](_))
+      val name: String = "alternative"
+      val bases: Seq[(String, RuleSet)] = Nil
+      val parents: Seq[RuleSet] = Seq(monoidK[A], applicative[A,B,C])
+      val props: Seq[(String, Prop)] = Seq(
+        "left distributivity" -> forAll((fa1: F[A], fa2: F[A], fab: A => B) =>
+          laws.alternativeLeftDistributivity[A,B](fa1, fa2, fab)),
+        "right distributivity" -> forAll((fa: F[A], fab1: F[A => B], fab2: F[A => B]) =>
+          laws.alternativeRightDistributivity[A,B](fa, fab1, fab2)),
+        "right absorption" -> forAll((fab: F[A => B]) =>
+          laws.alternativeRightAbsorption[A,B](fab))
```

Comment:
Yeah I agree. For some reason it stopped compiling until I did that. I didn't look into it much, and it's possible that I did something silly.